test/parallel/test-cluster-worker-exit.js

```diff
@@ -52,7 +52,6 @@ if (cluster.isWorker) {
     worker_emitDisconnect: [1, "the worker did not emit 'disconnect'"],
     worker_emitExit: [1, "the worker did not emit 'exit'"],
     worker_state: ['disconnected', 'the worker state is incorrect'],
-    worker_suicideMode: [false, 'the worker.suicide flag is incorrect'],
```

Comment:
just to confirm, this mode was only entered when using the deprecated API correct?